libp2p-pki/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "libp2p-pki"
```

Comment:
I don't think this should be named `libp2p-pki` but just `pki` instead, as it doesn't do anything specific to libp2p.